subiquity/common/apidef.py

```diff
@@ -290,6 +290,10 @@ class edit_partition:
                 def POST(data: Payload[ModifyPartitionV2]) \
                     -> StorageResponseV2: ...
 
+    class drivers:
+        def GET(wait: bool = False) -> Optional[bool]: ...
```

Comment:
I think we would benefit from injecting the value of `install` / `do_install` into the response, so that we can save/restore the status of the checkbox "install the drivers?" after unchecking it and hitting back.

Actually, I was thinking about something like:

```python
class drivers:
    def GET() -> bool:
        """ Return the value of do_install (aka. install drivers if available) """
    def POST(install: bool) -> None:
        """ Set the value of do_install """

    class available:
        def GET(wait: bool = False) -> Optional[bool]:
            """ Return whether drivers are available for installation. None means retry later.
            In the future, it could also be a list of drivers available.
            """ 
```

What is your opinion on this?

Reply:
Yes certainly the users choice should be sticky. Seems a bit silly to require two api calls but well. Details.

Reply:
Having both information in a single endpoint is fine as well IMO.